lib/main.dart

```diff
@@ -128,4 +128,129 @@ class _HomePageState extends State<HomePage> {
       },
     );
   }
+
+  void routeToInfo(ObjectInfo picData) {
+    setState(() {
+      Navigator.push(
+        context,
+        MaterialPageRoute(builder: (context) => ArtworkInfoScreen(picData)),
+      );
+    });
+  }
+}
+
+class ArtworkInfoScreen extends StatelessWidget {
+  final ObjectInfo picData;
+
+  ArtworkInfoScreen(this.picData);
+
+  @override
+  Widget build(BuildContext context) {
+    final imageData = (picData.images == null) ? null : picData.images[0];
+    final artistData = (picData.people == null) ? null : picData.people[0];
+    return Scaffold(
+      backgroundColor: Colors.teal[100],
+      appBar: AppBar(
+        title: Text('Artwork Information'),
+        backgroundColor: Colors.teal[50],
+      ),
+      body: SafeArea(
+        child: Center(
```

Comment:
`Center`, `Column`, `Expanded`, and `Padding` seem to be a bit extra here. Is there a reason why all of these widgets precede the ListView?